include/realizations/catchment/Bmi_C_Adapter.hpp

```diff
@@ -29,22 +24,30 @@ namespace models {
 
         public:
 
-            explicit Bmi_C_Adapter(std::string forcing_file_path, bool model_uses_forcing_file, bool allow_exceed_end,
+            explicit Bmi_C_Adapter(std::string library_file_path, std::string forcing_file_path,
```

Comment:
Are we assuming that BMI formulations will be reading forcings directly from disk?  I  was  under the impression that NGEN would read a  standard forcing and proxy the data to the formulations.

Reply:
We are assuming that in general, both are possible depending on the model.  This is based on my interpretation of [the `initialize` documentation](https://bmi.readthedocs.io/en/latest/#initialize) and follow-up discussions with Scott.  

The initial CFE BMI implementation was built to read its own data, at least initially.  That may need to change in the future, though.  Support for the proxying-type situation is being added via #219.